src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements Description, GetArea {
+    private int radius;
+    private String name = "circle";
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public int getRadius() {
+        return radius;
+    }
+
+    @Override
+    public void getDescription() {
+        System.out.println("Name: " + name + " radius: " + getRadius() + " units "
```

Comment:
```suggestion
        System.out.println("Name: circle " + " radius: " + getRadius() + " units "
```
and let's remove field `private String name = "circle";` from the class
and do the same for all other figures